packages/cloudflare/src/cli/commands/deploy.ts

```diff
@@ -27,11 +27,13 @@ export async function deployCommand(args: WithWranglerArgs<{ cacheChunkSize: num
 
 	const wranglerConfig = readWranglerConfig(args);
 
-	const envVars = await getEnvFromPlatformProxy({
+	const platformProxyEnvVars = await getEnvFromPlatformProxy({
 		configPath: args.configPath,
 		environment: args.env,
 	});
 
+	const envVars = { ...platformProxyEnvVars, ...process.env };
```

Comment:
`upload.ts` should be updated as well.

May be we could add a `includeProcessEnv: Boolean` to `getEnvFromPlatformProxy` so that we could document the purpose on the JSDoc of that functon.

I'm wondering if it should `{ ...platformProxyEnvVars, ...process.env }` or `{ ...process.env, ...platformProxyEnvVars}` - I would tend to prefer the latest as proxy vars are more specific as they could change with the environment but I don't have a very strong opinion on that. What do you think?

Reply:
> What about renaming the function getEnvironmentVariables and have it always include values from both platformProxy and process.env? would there be an issue with that?